bootstrap/handlers/clients.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/edgexfoundry/go-mod-registry/v3/pkg/types"
 	"github.com/edgexfoundry/go-mod-registry/v3/registry"
 
+	bscfg "github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/config"
```

Comment:
Alias not needed
```suggestion
	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/config"
```

Reply:
fine with me, just meant i had to refactor the existing 'config' variable. done